scrapers/Motherless.yml

```diff
@@ -0,0 +1,26 @@
+name: "Motherless"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - motherless.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+        $meta: //div[@class='media-meta']
+    scene:
+      Title: $meta//h1/text()
+      Date:
+        selector: $meta//div[@class='media-meta-stats']/span[not(contains(.,'Views')) and not(contains(.,'Favorites'))]/text()
```

Comment:
Recent videos have `1 d ago` or `xx m ago` but nothing can be done for that so good enough

Reply:
Yeah I noticed that was well, but like you said, nothing can be done for that currently.